src/dotnet/commands/dotnet-build/IncrementalPreconditions.cs

```diff
@@ -64,7 +64,7 @@ private string PreconditionsMessage()
             log.Append(
                 "Incremental compilation will be automatically enabled if the above mentioned project properties are not used. " +
                 "For more information on the properties and how to address them, please consult:\n" +
-                @"https://github.com/dotnet/cli/blob/master/Documentation/addressing-incremental-compilation-warnings.md");
+                @"https://github.com/dotnet/cli/blob/rel/1.0.0/Documentation/addressing-incremental-compilation-warnings.md");
```

Comment:
Hmm, this one concerns me.  I don't think we should be compiling hard-coded URLs in our code, especially pointing to branches in github.

@blackdwarf @cdmihai @piotrpMSFT - thoughts?  Should we make an fwlink for this?